core/snippets/snippets_parser.py

```diff
@@ -48,10 +49,12 @@ def create_snippets(documents: list[SnippetDocument]) -> list[Snippet]:
 
 
 def create_snippet(
-    document: SnippetDocument, default_context: SnippetDocument
+    document: SnippetDocument,
+    default_context: SnippetDocument,
 ) -> Snippet | None:
     snippet = Snippet(
-        name=document.name or default_context.name,
+        name=document.name or default_context.name or "",
```

Comment:
Am I missing something here? According to the documentation the name is required but you have an option for no name?

Reply:
It's just that the type doesn't support None. We are validating that there is a name a few lines below
https://github.com/talonhub/community/blob/c9a55b56f00851b826df4a05ca88cefd4a2898b9/core/snippets/snippets_parser.py#L74-L76